digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -7,23 +7,32 @@
 import com.treasuredata.client.TDClientHttpConflictException;
 import com.treasuredata.client.TDClientHttpNotFoundException;
 import com.treasuredata.client.TDClientHttpUnauthorizedException;
-import com.treasuredata.client.model.TDExportJobRequest;
+import com.treasuredata.client.model.TDJob;
 import com.treasuredata.client.model.TDJobRequest;
-import com.treasuredata.client.model.TDSavedQueryStartRequest;
+import com.treasuredata.client.model.TDJobSummary;
 import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigElement;
 import io.digdag.client.config.ConfigException;
+import io.digdag.spi.TaskExecutionException;
 import io.digdag.util.RetryExecutor;
 import io.digdag.util.RetryExecutor.RetryGiveupException;
 
 import java.io.Closeable;
-import java.time.Instant;
-import java.util.Date;
+import java.util.UUID;
 
+import static com.treasuredata.client.model.TDJob.Status.SUCCESS;
 import static io.digdag.util.RetryExecutor.retryExecutor;
 
 public class TDOperator
         implements Closeable
 {
+    private static final String JOB_ID = "job_id";
```

Comment:
Oh, you took care of the backward compatibility, nice!
https://github.com/treasure-data/digdag/pull/201/files#diff-9d50bfb9c6abc38992759addd61c9eb0R237